streamer/pipeline_configuration.py

```diff
@@ -307,6 +305,10 @@ class PipelineConfig(configuration.Base):
 
 
   def __init__(self, *args) -> None:
+    self.__class__.resolutions.default=[ # type: ignore
```

Comment:
Since resolution and channel layout have a behavior of "use every option that doesn't upscale/upmix", perhaps the best default would be the entire list.  Can you please try that and let me know how it goes?  Also, please update the docstrings for those fields to say that they default to all defined values from the bitrate config.